Mathlib/Algebra/Polynomial/Roots.lean

```diff
@@ -9,6 +9,7 @@ import Mathlib.Data.Set.Finite.Lemmas
 import Mathlib.RingTheory.Coprime.Lemmas
 import Mathlib.RingTheory.Localization.FractionRing
 import Mathlib.SetTheory.Cardinal.Order
+import Mathlib.Algebra.CharP.Basic
```

Comment:
not needed now

Reply:
I've applied this comment on a new PR as I no longer have write access to this branch: #26224